### PR TITLE
Remove useless docs

### PR DESCRIPTION
**Pull request by sy-records**
Seems to be #612 forgot to delete

### CODE REVIEW

**Comment by Girgias** [comment]
I didn't remove the notes in language-snippets so this could be rebased for that removal.

**Comment by sy-records** [comment]
You have removed the fgsql documentation, so its configuration items for these should be useless

File: features/persistent-connections.xml [hunk 1]
```diff
@@ -122,8 +122,7 @@
    behavior!
   </simpara>
   <para>
-   See also <function>fbsql_pconnect</function>,
-   <function>ibase_pconnect</function>, <function>ifx_pconnect</function>,
+   See also <function>ibase_pconnect</function>, <function>ifx_pconnect</function>,
    <function>ingres_pconnect</function>,
    <function>msql_pconnect</function>, <function>mssql_pconnect</function>,
```
Comment (Girgias) [inline features/persistent-connections.xml:127]: Aren't ingress and msql docs also removed?

Comment (kamil-tekiela) [reply features/persistent-connections.xml:127]: Screenshot_20210823-133820
 A lot of them are gone now.

Comment (sy-records) [reply features/persistent-connections.xml]: Removed

**Pull request merged**